app/pages/lab-pages-editor/components/TasksPage/components/EditStepDialog/EditStepDialog.jsx

```diff
@@ -4,10 +4,17 @@ import PropTypes from 'prop-types';
 import EditTaskForm from './EditTaskForm.jsx';
 import CloseIcon from '../../../../icons/CloseIcon.jsx';
 
+const taskNames = {
+  'drawing': 'Drawing',
+  'single': 'Single Question',
+  'text': 'Text',
+}
+
 function EditStepDialog({
   allTasks = {},
```

Comment:
this is getting really nit-picky and very personal style... But I like putting the primary / common noun first. For example, `tasksAll` and `taskUpdate`. This makes browsing through the code a bit easier to understand.